HW_3/templates/register.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<h1>Login</h1>
+<form method="POST" action="{{ url_for('register') }}">
```

Comment:
Параметр `action` не нужен. за это отвечает функция представления
```suggestion
<form method="POST">
```